docs/src/examples/components/Menu/Usages/MenuExampleWithSubmenuControlled.shorthand.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { Menu } from '@stardust-ui/react'
+
+class MenuWithSubmenuControlledExample extends React.Component {
+  state = { editorialsMenuOpen: false }
+
+  handleMenuOpenChange = (e, { menuOpen }) => {
+    this.setState({ editorialsMenuOpen: menuOpen })
+  }
+
+  renderItems = () => {
+    return [
+      {
+        key: 'editorials',
+        content: 'Editorials',
+        menuOpen: this.state.editorialsMenuOpen,
+        onMenuOpenChange: this.handleMenuOpenChange,
+        menu: {
+          items: [
+            { key: '1', content: 'item1' },
+            {
+              key: '2',
+              content: 'item2',
+              menu: [{ key: '1', content: 'item2.1' }, { key: '2', content: 'item2.2' }],
+            },
+            {
+              key: '3',
+              content: 'item3',
+              menu: [{ key: '1', content: 'item3.1' }, { key: '2', content: 'item3.2' }],
+            },
```

Comment:
I suggest to make it more compact:

```
menu: [
  { key: '1', content: 'item1' },
  { key: '2', content: 'item2' },
  { key: '3', content: 'item3' },
],
```

Reply:
As it will become more compact, do we need `renderItems()` at all?

Reply:
I would leave `renderItems()`, for better readability